operations/iuf/workflows/configuration.md

```diff
@@ -0,0 +1,183 @@
+# Configuration
+
+This section ensures product configuration have been defined, customized, and is available for later steps in the workflow.
+
+This workflow uses `${ADMIN_DIR}` to retain files that define site preferences for IUF. `${ADMIN_DIR}` is defined separately from `${ACTIVITY_DIR}` and `${MEDIA_DIR}` based on the assumption that the files in `${ADMIN_DIR}` will be
+used when performing future IUF operations unrelated to this workflow.
+
+**`NOTE`** The following steps assume `${ADMIN_DIR}` is empty. If this is not the case, i.e. `${ADMIN_DIR}` has been populated by previous IUF workflows, ensure the content in `${ADMIN_DIR}` is up to date with the latest content
+provided by the HPC CSM Software Recipe release content being installed. This may involve merging new content provided in the latest branch of the `hpc-csm-software-recipe` repository in VCS with the existing content in `${ADMIN_DIR}`.
```

Comment:
```
This may involve merging new content provided in the latest branch of the `hpc-csm-software-recipe` repository in VCS with the existing content in `${ADMIN_DIR}`.
```

perhaps

```
This may involve merging new content provided in the latest branch of the `hpc-csm-software-recipe` repository in VCS or from the files extracted from the HPC CSM Software Recipe with the existing content in `${ADMIN_DIR}`.
```

Reply:
Fixed.